steering/CHARTER.md

```diff
@@ -55,8 +49,14 @@ modifications to the Steering Committee Charter.
     three Companies, then the third highest Company shall be awarded a seat,
     and then the formula and procedure shall be applied to eight seats instead
     of nine.
+1. Contribution Seats will be calculated on February 1 each year, based on the
+agreed formula and procedure for the preceding 12 months. Contribution Seats
+shall become active at the conclusion of the Community Seat election which
+immediately follows their allocation.
+1. If necessary, a company holding a Contribution Seat may change the appointed
+individual at any time during the term.
 1. There shall be four **Community Seats** elected by the Istio contributors
-and community, beginning in August 2020.
+and community.
```

Comment:
What does this mean?  What community members will be allowed to vote that are not contributors?

Reply:
i think only the ones who have 1 PR merged in past 12 months are allowed to vote

Reply:
Who can vote it set out elsewhere:

- People with the right number of merged PRs, as Lin says
- Those who have applied for, and been granted, an exception.

The only change here is removing the date we started the charter.